resources/views/meetups/create.blade.php

```diff
@@ -1,42 +1,102 @@
 @extends('layouts.app')
 
 @section('content')
-    <div>
-        <div>
-            <h1>Create Meetup</h1>
-            @auth
-                <form action="{{ route('meetups.store') }}" method="post">
-                    @csrf
+    <div class="container md:w-[800px] mx-auto">
+        @auth
+            <form action="{{ route('meetups.store') }}" method="post" class="bg-white p-6 mt-20 rounded-20 shadow-xl">
+                @csrf
+                <div>
                     <div>
-                        <label for="title">Title:</label>
-                        <input type="text" id="title" name="title" value="{{ old('title') }}">
-                        <x-input-error for="title"/>
+                        <h3 class="text-xl leading-6 font-medium text-gray-900">
+                            Create Meetup
+                        </h3>
                     </div>
-                    <div>
-                        <label for="description">Description:</label>
-                        <textarea name="description" id="description" cols="30" rows="4">{{ old('description') }}</textarea>
-                        <x-input-error for="description"/>
-                    </div>
-                    <div>
-                        <label for="date">Date:</label>
-                        <input type="dateTime-local" id="date" name="date" value="{{ old('date') }}">
-                        <x-input-error for="date"/>
-                    </div>
-                    <div>
-                        <label for="place">Place:</label>
-                        <input type="text" id="place" name="place" value="{{ old('place') }}">
-                        <x-input-error for="place"/>
-                    </div>
-                    <div>
-                        <label for="language">Language:</label>
-                        <input type="text" id="language" name="language" value="{{ old('language') }}">
-                        <x-input-error for="language"/>
+                    <div class="mt-6 flex flex-col gap-7">
+                        <div class="sm:flex items-center">
+                            <img id="image" src="{{ asset('static/images/no_image.jpeg') }}" alt="meetup"
+                                class="w-full sm:w-[400px] h-[200px]">
+                            <input type="file" accept="image/*" id="image-input" class="hidden">
+                            <label for="image-input"
+                                class="inline-flex items-center px-4 py-2 border border-transparent shadow-sm text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 cursor-pointer mx-auto w-full justify-center mt-3 sm:w-auto sm:mt-0">
+                                <!-- Heroicon name: solid/mail -->
```

Comment:
Instead of long `svg`, import that package for `heroicons` - https://heroicons.dev/